examples/tomo/data/mrc_io.py

```diff
@@ -0,0 +1,102 @@
+# Copyright 2014-2016 The ODL development group
+#
+# This file is part of ODL.
+#
+# ODL is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Example for the usage of the MRC reader/writer.
+
+The code below first reads an existing MRC file from disk and prints
+some attributes read from the header. If possible, the data is displayed.
+Then, a new header is created using a utility function, and a new
+(temporary) file is written to disk.
+This simulates the start and end of a reconstruction workflow, where
+first projection data is read from disk, and finally the reconstructed
+volume is written back to disk, using a different header.
+"""
+
+from __future__ import print_function
+
+try:
+    import matplotlib.pyplot as plt
```

Comment:
Lets not have stuff like this, the whole library basically has matplotlib as a dependency, skipping it is if you truly want to go minimal.